src/ast/node.cpp

```diff
@@ -376,6 +377,10 @@ namespace phylanx { namespace ast
     std::ostream& operator<<(std::ostream& out, operand const& op)
     {
         out << "operand";
+        int ind = op.index();
+        if(ind == 1) {
+          auto ident = phylanx::util::get<1>(op.get());
+        }
```

Comment:
Shouldn't this print something for `ind == 2` as well? Wouldn't it be better to use visit()?

Reply:
Sorry, that was a mistake. I reverted it.